package.json

```diff
@@ -13,10 +13,10 @@
   "author": "Zach Bruggeman <zachb@diy.org>",
   "license": "MIT",
   "dependencies": {
-    "js-beautify": "^1.6.14",
+    "js-beautify": "^1.9.0",
     "optimist": "^0.6.1",
-    "walk": "^2.3.9",
-    "xhr": "^2.4.0"
+    "walk": "^2.3.14",
```

Comment:
Not necessarily relevant to this PR, but I don't see any use of `walk` outside of test code.  Probably belongs in dev deps?

Reply:
Yep.

Reply:
https://github.com/dogescript/dogescript/issues/206